help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+
+### Personalization consistency {#personalization-consistency}
+
+This connector should be used by marketing professionals aiming to deliver a consistent personalization across the site from home page to search.  
+
+For example, as a marketer, you might want to build rich audiences from multiple user data sources including Algolia that can be shared with various channels for targeting strategies, leading to a boost in campaign personalization and conversion.
+
+The use case is executed through a combination of both destination and source connectors.
+
+You would start by importing your existing [!DNL Algolia] user profiles into Adobe Experience Platform Real-time CDP and other sources to begin creating rich audiences with the source connector. Marketers would create audiences using the profile data that can be sent to Algolia for search and recommendation personalization.
+
+The customer would then use the corresponding [Algolia User Profiles](/help/sources/connectors/data-partners/algolia-user-profiles.md) source to ingest and augment customer profiles back into Adobe Real-Time CDP.
+
+## Prerequisites {#prerequisites}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+## Supported audiences {#supported-audiences}
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description |
+|---------|---------|----------|
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md). |
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+|---------|----------|---------|
+| Export type | **[!DNL Profile request]** | You are requesting all the audiences that are mapped in the Algolia User Segments destination for a single profile.|
```

Comment:
Is this a "Profile request" destination? Profile request describes destination platforms which _retrieve_ the data from Platform themselves, instead of _receiving_ it from Platform. I'm not sure this is the case here.

I believe this should be Audience Export (https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#export-type-frequency).